src/Service/PdfCreator.php

```diff
@@ -4,38 +4,39 @@
 
 namespace App\Service;
 
-use Spipu\Html2Pdf\Html2Pdf;
+use App\ValueObject\PdfFormat;
+use App\ValueObject\PdfOrientation;
+use Dompdf\Dompdf;
+use Dompdf\Options;
 
 final class PdfCreator implements PdfCreatorInterface
 {
     private $pdf;
     private $orientation;
     private $format;
-    private $lang;
-    private $unicode;
-    private $encoding;
-    private $margin;
 
-    public function __construct(string $orientation, string $format, string $lang, bool $unicode, string $encoding, array $margin)
+    public function __construct(PdfOrientation $orientation, PdfFormat $format)
     {
-        $this->orientation = $orientation;
-        $this->format = $format;
-        $this->lang = $lang;
-        $this->unicode = $unicode;
-        $this->encoding = $encoding;
-        $this->margin = $margin;
+        $this->orientation = $orientation->getValue();
+        $this->format = $format->getValue();
     }
 
     public function create(): void
     {
-        $this->pdf = new Html2Pdf($this->orientation, $this->format, $this->lang, $this->unicode, $this->encoding, $this->margin);
-        $this->pdf->setTestIsImage(false);
+        $this->pdf = new Dompdf(new Options([
+            'isHtml5ParserEnabled' => true,
+            'isRemoteEnabled' => true,
+        ]));
+        $this->pdf->setPaper($this->format, $this->orientation);
```

Comment:
I'd probably defer using `->getValue()` to this line, keeping them objects this far